flask_monitoringdashboard/database/data_base_queries/mongo_db_objects.py

```diff
@@ -26,19 +27,59 @@ def _safe_mongo_call(*args, **kwargs):
     return _safe_mongo_call
 
 
+def safe_mongo_call_find(call, call_type):
+    """
+    In order to minimize the risk of failure and the impact on the app performance,
+    we need to allow the application to be able to read from the secondaries.
+    In such cases, we will have an eventual data consistence. Hence, we should re-try to read
```

Comment:
I don't understand this comment. 
Does the MongoDB driver let you manually query replicas? 
How do you exactly do this?

Reply:
To add a bit of context, you can add those configuration the the URI connection string and the driver pick them up from there.